services/app-web/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -99,119 +99,127 @@ export const Dashboard = (): React.ReactElement => {
 
     return (
         <>
-            <GridContainer
-                className={styles.container}
-                data-testid="dashboard-page"
-            >
-                {programs.length ? (
-                    <section className={styles.panel}>
-                        {justSubmittedSubmissionName && (
-                            <SubmissionSuccessMessage
-                                submissionName={justSubmittedSubmissionName}
-                            />
-                        )}
-                        <div className={styles.panelHeader}>
-                            <h2>Submissions</h2>
-                            <div>
-                                <Link
-                                    asCustom={NavLink}
-                                    className="usa-button"
-                                    variant="unstyled"
-                                    to={{
-                                        pathname: '/submissions/new',
-                                    }}
-                                >
-                                    Start new submission
-                                </Link>
+            <div className={styles.wrapper}>
```

Comment:
Oh! This would be useful on other full width pages as well. 🤔  We don't have a pattern around this yet - an element that fills over the background of the app - and then you lay the grid on top. I feel like Landing page could use this too.

Would you mind pulling this out as a component?  `FullWidthPageWrapper` or something like that? It could takes all the props a div `JSX.IntrinsicElements['div']`  takes - that way anyone using it could override the styles

Reply:
oh I'm too late nvm lol. We can do this in a future PR